.azure-pipelines/azure-pipelines-osx.yml

```diff
@@ -42,6 +42,10 @@ jobs:
       setup_conda_rc ./ ./recipes ./.ci_support/${CONFIG}.yaml
 
       source run_conda_forge_build_setup
+
+      conda install -c defaults conda=4.7.5 --yes --quiet
+      conda config --env --add pinned_packages conda=4.7.5
+
```

Comment:
OSX hack required because of https://github.com/conda/conda/issues/9845

Reply:
@dhirschfeld, lets just disable the osx build instead. we will fix these things in bulk once we find a fix.